src/components/Footer/Footer.js

```diff
@@ -201,14 +215,14 @@ export default function Footer() {
                             </a>
                         </li>
                         <li className={classes.iconLi}>
-                            <a href="#">
+                            <a href="https://www.instagram.com/civictechto/?hl=en">
                                 <Avatar className={classes.iconAvatar}>
                                     <InstagramIcon />
                                 </Avatar>
                             </a>
                         </li>
                         <li className={classes.iconLi}>
-                            <a href="https://twitter.com/?lang=en">
+                            <a href="https://twitter.com/CivicTechTO?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor">
```

Comment:
Can this be changed to : [https://twitter.com/CivicTechTO](https://twitter.com/CivicTechTO)

Reply:
fixed